ui/css/oae.signup.css

```diff
@@ -0,0 +1,310 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+main > h3 {
+    border-bottom: 1px #EEEEEE solid;
+    margin-left: 20px;
+    margin-right: 20px
+}
+
+#signup-options-container {
+    display: -webkit-flex;
```

Comment:
The 100% height issue makes what we can do with floats very limiting. I can't see a way to do it w/ bootstrap grids.